app/Http/Controllers/FaqController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Faq;
+use Illuminate\Http\Request;
+
+class FaqController extends Controller
+{
+    public function create()
```

Comment:
Remember to follow op on these automatically generated PHPcs comments. In the exam, there might be a point deduction when we find them in your code.

Reply:
I repeat: In the exam, there might be a point deduction when we find them in your code. Ask a TA or teacher if you can't see these comments yourself.